hbase-asyncfs/src/test/java/org/apache/hadoop/hbase/security/HBaseKerberosUtils.java

```diff
@@ -172,23 +170,6 @@ public static void setSSLConfiguration(HBaseCommonTestingUtil utility, Class<?>
     KeyStoreTestUtil.setupSSLConfig(keystoresDir.getAbsolutePath(), sslConfDir, conf, false);
   }
 
-  public static UserGroupInformation loginAndReturnUGI(Configuration conf, String username)
```

Comment:
It's not clear to me why HBaseKerberosUtils in in the hbase-asyncfs module in the first place.

IMO we should consider moving the whole class to hbase-commons. (not necessarily in this ticket)